target/linux/ath79/dts/qca9563_tl-wr1043nd-v4.dts

```diff
@@ -10,3 +10,16 @@
 	compatible = "tplink,tl-wr1043nd-v4", "qca,qca9563";
 	model = "TP-Link TL-WR1043ND Version 4";
 };
+
+&usb_phy0 {
+	status = "okay";
+};
+
+&usb0 {
+	status = "okay";
+
+	hub_port0: port@1 {
```

Comment:
What about the trigger-sources of usb led? This will throw an error if anyone will use qca9563_tl-wr1043n.dtsi and not have hub_port0.

Reply:
you are right